backend/app.py

```diff
@@ -10,6 +11,19 @@
 
 app.mount("/static", StaticFiles(directory="static"), name="static")
 
+origins = [
+    'http://ui.oasis.lvh.me:3000',
+    'http://localhost:3000'
```

Comment:
For now it's ok, but eventually we'll need ENV variables for both origins, right?

Reply:
Not really - this are only the dev access points. On prod/stg, i believe the origin will be the same as the react app (since it will be compiled & mounted onto the python server!)